cpp/src/barretenberg/grumpkin_srs_gen/grumpkin_srs_gen.cpp

```diff
@@ -7,6 +7,7 @@
 #include "barretenberg/common/net.hpp"
 
 const std::string protocol_name = "BARRETENBERG_GRUMPKIN_IPA_CRS";
+// WORKTODO: this is incorrect now right?
```

Comment:
what is the difference between WORKTODO and a regular TODO?

Reply:
WORKTODO is something I search for and then resolve before putting the PR up for review.

Reply:
Tbh maybe I should use something shorter like `HEY`

Reply:
Or perhaps `HMM` alla Zac